packages/grid/x-data-grid/src/models/gridFilterModel.ts

```diff
@@ -8,11 +8,21 @@ export interface GridFilterModel {
    * @default []
    */
   items: GridFilterItem[];
-
   /**
    * - `GridLinkOperator.And`: the row must pass all the filter items.
    * - `GridLinkOperator.Or`: the row must pass at least on filter item.
    * @default `GridLinkOperator.Or`
    */
   linkOperator?: GridLinkOperator;
+  /**
+   * values used to quick filter rows
+   * @default `[]`
+   */
+  quickFilterValues?: any[];
+  /**
+   * - `GridLinkOperator.And`: the row must pass all the values.
+   * - `GridLinkOperator.Or`: the row must pass at least one value.
+   * @default `GridLinkOperator.Or`
```

Comment:
The value in `getDefaultGridFilterModel` is "AND". One of the places is wrong.

```suggestion
   * @default `GridLinkOperator.And`
```